scalastyle-config.xml

```diff
@@ -150,6 +150,11 @@ This file is divided into 3 sections:
       // scalastyle:on println]]></customMessage>
   </check>
 
+  <check customId="invalidMDC" level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
+    <parameters><parameter name="regex">s".*MDC</parameter></parameters>
```

Comment:
This regex looks too broad to me. For example, this will block the following, right?
```
s"text and no MDC from Spark will print $msg"
```

Reply:
You are right. I updated it to match the patther of `s"...${MDC(...)}`